### PR TITLE
Fixes #1334:  visited link should not change color on homepage

### LINKED ISSUE #1334: Visited link should not change color on homepage.
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

The color of visited links changes on homepage.
**Expected behavior:**

It should not change on homepage. Compare it with Google.
**Screenshots of the issue:**

image

### PR DESCRIPTION
**Pull request by rishavpandey43**
Fixes #1334 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- I've fixed the issue of changing of color of link visited in homepage

- https://pr-1338-fossasia-susper.surge.sh

-

### CODE REVIEW

**Comment by rishavpandey43** [comment]
@shaharyar-shamshi 
changed the code from style.css to css/base.css

**Comment by rishavpandey43** [comment]
@shaharyar-shamshi 
Thanks a lot for your review, since I'm new in the open source contribution, I faced some issue to organise the code in proper way.
Hope to see your help in coming future, if required.

**Comment by shreyanshdwivedi** [comment]
@rishavpandey43 I think u wanted to tag me :sweat_smile: 
I'll be there to help. Btw please squash your commits

File: src/.fuse_hidden0000271c00000002 [hunk 1]
```diff
@@ -0,0 +1 @@
+/* You can add global styles to this file, and also import other style files */
```
Comment (rajvaibhavdubey) [inline src/.fuse_hidden0000271c00000002]: I don't think that this file is needed.

Comment (rishavpandey43) [reply src/.fuse_hidden0000271c00000002]: This file was added by default, I've even removed it.

**Comment by rishavpandey43** [comment]
@shreyanshdwivedi @rajvaibhavdubey 
Thank you'll for review, I've removed the extra file and also squashed the commits.
Please, have a look once again and suggest if any change is needed more.

**Comment by mariobehling** [comment]
Closing due to inactivity.

**Pull request closed without merging**